recipes/TIES20/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "TIES20" %}
+{% set version = "20.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/UCL-CCS/TIES20/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 4af8f09bc480467ef8941dbc4f1a96edadb1dad6111929fc1b664eb67d266ef7
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  noarch: python
+  entry_points:
+    - ties = ties:cli.command_line_script
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```

Comment:
Noarch implies no compiled code. You need to choose between noarch OR compiled cython modules.

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Reply:
Thanks, we'll remove the "noarch: python".